docs/index.yaml

```diff
@@ -1,9 +1,37 @@
 apiVersion: v1
 entries:
   illumidesk:
+  - apiVersion: v2
+    appVersion: 0.1.1
+    created: "2020-12-10T19:12:39.811233-05:00"
+    dependencies:
+    - import-values:
+      - child: rbac
+        parent: rbac
+      name: jupyterhub
+      repository: https://jupyterhub.github.io/helm-chart/
+      version: 0.10.2
+    - condition: postgresql.enabled
+      name: postgresql
+      repository: https://charts.bitnami.com/bitnami
+      version: 10.1.3
+    - condition: datadog.enabled
+      name: datadog
+      repository: https://helm.datadoghq.com
+      version: 2.4.29
+    description: An extention of the JupyterHub chart with additional IllumiDesk resources
+    digest: 45b66907cf3dbe138f606a4e882b6190b95389bc4516fc59af4caaef37932ee6
+    icon: https://configs.illumidesk.com/images/illumidesk-80.png
+    maintainers:
+    - email: hello@illumidesk.com
+      name: IllumiDesk Team
+    name: illumidesk
+    urls:
+    - illumidesk-0.1.1.tgz
+    version: 0.1.1
   - apiVersion: v2
     appVersion: 0.1.0
```

Comment:
Shouldn't this be 0.1.1?

Reply:
older versions of the charts are still available but I added 0.1.1